browser-test/src/static_text.test.ts

```diff
@@ -41,6 +57,24 @@ describe('Static text question for applicant flow', () => {
     await applicantQuestions.seeStaticQuestion(staticText)
   })
 
+  it('displays the accordion', async () => {
+    const {page, applicantQuestions} = ctx
+    await applicantQuestions.applyProgram(programName)
+
+    expect(await page.innerHTML('.cf-applicant-question-text')).toContain(
+      'cf-accordion',
+    )
+  })
+
+  it('expands accordion when accordion header clicked', async () => {
```

Comment:
suggestion (non-blocking): combine this test with `validate screenshot when accordion open`

Reply:
Good idea.  Doing that now.